packages/neos-ui-redux-store/src/UI/ContentCanvas/index.js

```diff
@@ -13,6 +13,7 @@ const SET_PREVIEW_URL = '@neos/neos-ui/UI/ContentCanvas/SET_PREVIEW_URL';
 const SET_SRC = '@neos/neos-ui/UI/ContentCanvas/SET_SRC';
 const FORMATTING_UNDER_CURSOR = '@neos/neos-ui/UI/ContentCanvas/FORMATTING_UNDER_CURSOR';
 const SET_CURRENTLY_EDITED_PROPERTY_NAME = '@neos/neos-ui/UI/ContentCanvas/SET_CURRENTLY_EDITED_PROPERTY_NAME';
+const DOCUMENT_INITIALIZING = '@neos/neos-ui/UI/ContentCanvas/DOCUMENT_INITIALIZING';
```

Comment:
I mean in redux originally actions are not events but rather commands in CQRS language, but we use it for both, regarding the context.